samples/ScheduleDemo/Program.cs

```diff
@@ -0,0 +1,153 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+using Microsoft.DurableTask.Client;
+using Microsoft.DurableTask.Client.AzureManaged;
+using Microsoft.DurableTask.ScheduledTasks;
+using Microsoft.DurableTask.Worker;
+using Microsoft.DurableTask.Worker.AzureManaged;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+
+// Create the host builder
+IHost host = Host.CreateDefaultBuilder(args)
+    .ConfigureServices(services =>
+    {
+        string connectionString = Environment.GetEnvironmentVariable("DURABLE_TASK_SCHEDULER_CONNECTION_STRING")
+            ?? throw new InvalidOperationException("Missing required environment variable 'DURABLE_TASK_SCHEDULER_CONNECTION_STRING'");
+
+        // Configure the worker
+        _ = services.AddDurableTaskWorker(builder =>
+        {
+            // Add the Schedule entity and demo orchestration
+            builder.AddTasks(r =>
+            {
+                // Add a demo orchestration that will be triggered by the schedule
+                r.AddOrchestratorFunc<string, string>("DemoOrchestration", async (context, symbol) =>
```

Comment:
I suggest you create this orchestration as a class rather than a lambda. That will make the sample much easier to read.